code/modules/awaymissions/snpc.dm

```diff
@@ -4,7 +4,7 @@
 	var/squad_member = 0				// was spawned by squad
 	var/home_z
 
-/mob/living/carbon/human/interactive/away/New()
+/mob/living/carbon/human/interactive/away/Initialize(mapload)
```

Comment:
Hmm, why this change? It's already in my maploader fix PR

Reply:
Mostly because I was getting lots of runtimes in my testing; I tried to test species wherever I could, up to, and including interactive humans--and since I had to change one of the interactive mob's anyway (to set the species to skrell, properly), it seemed a bit silly to not fix the others.